app/assets/stylesheets/editor.scss

```diff
@@ -17,6 +17,7 @@
 }
 
 .fullscreen {
+  height: 82vh !important;
```

Comment:
This causes all `.fullscreen` elements to get this height, overriding the value set in the `base.scss` file.